docs/kubernetes/pouch_with_kuberneters_deploying.md

```diff
@@ -0,0 +1,152 @@
+# Pouch with kuberneters deploying
```

Comment:
s/kuberneters/Kubernetes. 

Kubernetes is a specified noun, so we need to uppercase the first letter every place where it appears.